cmd/podman/system/prune.go

```diff
@@ -55,22 +55,18 @@ func init() {
 
 func prune(cmd *cobra.Command, args []string) error {
 	var err error
-
 	// Prompt for confirmation if --force is not set
 	if !force {
 		reader := bufio.NewReader(os.Stdin)
 		volumeString := ""
 		if pruneOptions.Volume {
 			volumeString = `
-        - all volumes not used by at least one container`
+	- all volumes not used by at least one container`
 		}
-		fmt.Printf(`
-WARNING! This will remove:
-        - all stopped containers%s
-        - all stopped pods
-        - all dangling images
-        - all build cache
-Are you sure you want to continue? [y/N] `, volumeString)
+
+		message := fmt.Sprintf(createPruneWarningMessage(pruneOptions), volumeString, "Are you sure you want to continue? [y/N] ")
+		fmt.Printf("%s", message)
```

Comment:
Nit. You can use Printf directly. No need to call both Sprintf and Printf.

Reply:
done